### ISSUE TITLE
Ctrl+B find nothing

### ISSUE THREAD

**Issue by sundayJJ** [x64dbg/x64dbg#3443]
### Operating System

Win10

### x64dbg Version

2024-09-10_15-35

### Describe the issue

Ctrl+B in [Memory Map]
cann't find any thing if no check [Entire Block]

### Steps to reproduce

1. Ctrl+B in [Memory Map]
2. cann't find any thing if no check [Entire Block]
3.

**Comment by mrexodia** [comment]
It starts searching from your selection, Entire Block will search the entire memory.

**Comment by wonderkun** [comment]
I  use  Ctrl+A to select all before find pattern in memory map. But I can find nothing.

**Comment by wonderkun** [comment]
> I use Ctrl+A to select all before find pattern in memory map. But I can find nothing.

on wow64 process!.

**Comment by G3ph4z** [comment]
I believe it can be closed, as it seems that it is not a bug, but a feature.

**Comment by mrexodia** [comment]
Yeah it is a feature, but I think the way the 'entire block' checkbox is labeled is very unclear. Probably it's better to switch to a radio box with two options 'Search all memory' and 'Start search from <address>'. I will change the labels to better reflect this.

**Comment by G3ph4z** [comment]
That makes sense, definitely feels more _clear_ than the current solution.

**Comment by sundayJJ** [comment]
> Start search from <address>
I already know the reason.
it not "**_Start search from <address>_**", it only search the selected section.
you can try it

**Comment by G3ph4z** [comment]
@mrexodia I did some digging in the code and tested it a bit, and I think it searches in the selected region if the 'Entire Block' option is not checked.

**Comment by mrexodia** [comment]
Yeah seems like it's like @sundayJJ said. I think this behavior changed at some point with a PR, but probably the UI improvements could make this much clearer.

**Comment by G3ph4z** [comment]
> Yeah it is a feature, but I think the way the 'entire block' checkbox is labeled is very unclear. Probably it's better to switch to a radio box with two options `Search all memory` and `Start search from <address>`. I will change the labels to better reflect this. There is a similar issue with other search dialogs (for instructions etc).

I'm happy to do it if you don't mind. I know you're busy with CI and VS2022.
